tests/utils/test_client.py

```diff
@@ -39,6 +39,13 @@ def test_invalid_credentials():
         utils.client.create(**CONFIG["my_other_instance"], enforce_healthy=True)
 
 
+@mock_api()
+def test_store_auth_cookie():
+    my_client = utils.client.create(**CONFIG["my_instance_name"], store_auth_cookie=True)
+    assert my_client.session.cookies is not None
```

Comment:
```suggestion
    assert "authToken" in my_client.session.cookies.keys()
```
You should be a bit more specific here.  `my_client.session.cookies` is actually never `None`.  When no cookies are set it's still an empty `RequestsCookieJar`, so to make sure this specific cookies has been set you need to inspect the list returned by its `keys()` method.